bitnami/harbor/templates/chartmuseum/chartmuseum-dpl.yaml

```diff
@@ -41,6 +41,7 @@ spec:
         {{- end }}
     spec:
       {{- include "harbor.imagePullSecrets" . | nindent 6 }}
+      automountServiceAccountToken: false
```

Comment:
Would it have sense to make this configurable in the values.yml?

Reply:
harbor doesnt use any of those tokens and the official chart dropped them to improove security.
So im not sure that there is an usecase to have them in the containers, ever.

Reply:
Thanks for the clarification!